framework/doc/content/syntax/MeshModifiers/index.md

```diff
@@ -0,0 +1,18 @@
+# MeshModifiers
+
+The `MeshModifiers` system is designed to be able to modify the connectivity of the MooseMesh.
+
+## Postprocessing with MeshModifiers
+
+For now, the mesh divisions can only be output using a [MeshDivisionAux.md].
+We are planning to be able to compute averages, integrals, and various reductions
+on mesh divisions, please stay tuned.
+
+## Transferring with MeshModifiers
+
+This feature is a work in progress.
+
+!alert note
+Please let us know on [GitHub Discussions](https://github.com/idaholab/moose/discussions)
+how you are using the `MeshModifiers` system so we can include your techniques on this page!
```

Comment:
```suggestion
```
not relevant for mesh modifiers

see the syntax page for user objects please. there are some commands we want to add to list all the objects